src/spikeinterface/core/template_tools.py

```diff
@@ -31,7 +31,12 @@ def get_dense_templates_array(one_object: Templates | SortingAnalyzer, return_sc
             )
         ext = one_object.get_extension("templates")
         if ext is not None:
-            templates_array = ext.data["average"]
+            if "average" in ext.data:
+                templates_array = ext.data.get("average")
+            elif "median" in ext.data:
+                templates_array = ext.data.get("median")
+            else:
```

Comment:
One question:
Are percentiles where this else is triggered? 

One small suggestion:
my only suggestion which is very light would be call it `template_ext` instead of `ext` to be more explicit, but this is less crucial.

Reply:
I think I would have done the reverse.
In case we have both median and average which one do we want ? median maybe no ?

Reply:
I think the better case is to force the user to make the choice, but it comes back to ease of use vs maximal control. And I think this project has become big enough that more of the users want ease of use (ie intelligent defaults) vs needing to control everything. 

I'm curious what's your view on median vs average for this? I think average is used more commonly no? Although median is cool especially if you have some outlier artifact "spikes" I guess median would be better.